src/containers/layouts/ecommerce/productDetails/productDetails.component.tsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+import {
+  View,
+  ScrollView,
+  Image,
+  ViewProps,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import {
+  RadioGroup,
+  Radio,
+  RadioProps,
+  Button,
+  Input,
+} from '@kitten/ui';
+import { Text } from '@src/components/common';
+import {
+  Product as ProductModel,
+  Comment as CommentModel,
+} from '@src/core/model';
+import { CommentsList1 } from '@src/containers/layouts/messaging/commentsList1/commentsList1.component';
```

Comment:
should be `@src/containers/layouts/messaging` or moved to `common` or `messaging` components

Reply:
Refactored.